linked_list/linked_list.py

```diff
@@ -86,10 +149,22 @@ def visit(self):
 
     # method to reverse the singly linked list
     # note: the nodes should be moved and not just the values in the nodes
-    # Time Complexity: ?
-    # Space Complexity: ?
+    # Time Complexity: O(n)2
+    # Space Complexity: O(n)2
     def reverse(self):

```

Comment:
👍 , however the time/space complexities are O(n) for time and O(1) for space as you're not creating any new lists.